### PR TITLE
Documentation | Create documentation for annotation

### PR DESCRIPTION
**Pull request by stalberm**
## Description
Adds documentation for the annotation related code
<!--
## Related Issues
References #40

### CODE REVIEW

File: lib/screens/main/annotation.dart [hunk 1]
```diff
@@ -126,6 +144,7 @@ class _AnnotationPageState extends State<AnnotationPage> {
                 },
                 child: const Text("Complete Annotations"),
               ),
+              //
```
Comment (theusaf) [inline lib/screens/main/annotation.dart:147]: ?

Comment (bpugh5) [reply lib/screens/main/annotation.dart:147]: I second this, delete unused comment?

**Review by bpugh5 — DISMISSED** [review]
Looks good, see comments

**Review by theusaf — APPROVED** [review]
LGTM

**Review by kimberlymarkley — APPROVED** [review]

**Pull request merged**